org.sparkleshare.SparkleShare.json

```diff
@@ -0,0 +1,136 @@
+{
+    "app-id": "org.sparkleshare.Sparkleshare",
+    "branch": "master",
+    "command": "sparkleshare",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "metadata": "metadata",
+    "separate-locales": false,
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=system-bus",
+        "--socket=session-bus",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--share=ipc",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "mono",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.mono-project.com/sources/mono/mono-4.2.2.30.tar.bz2",
+                    "sha256": "57858cd033be9915d7abdc5158c1faae8fa05757c3b7117cab3d703aa696c56b"
+                }
+            ]
+        },
+        {
+            "name": "openssh",
+            "config-opts": ["--without-pie"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://mirror.exonetric.net/pub/OpenBSD/OpenSSH/portable/openssh-7.5p1.tar.gz",
+                    "sha256": "9846e3c5fab9f0547400b4d2c017992f914222b3fd1f8eee6c7dc6bc5e59f9f0"
+                }
+            ]
+        },
+        {
+            "name": "git",
+            "config-opts": ["--with-openssl=no"],
+            "make-args": ["NO_PERL=1"],
+            "make-install-args": ["NO_PERL=1"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.kernel.org/pub/software/scm/git/git-2.14.1.tar.xz",
+                    "sha256": "6f724c6d0e9e13114ab35db6f67e1b2c1934b641e89366e6a0e37618231f2cc6"
+                },
+                {
+                    "type": "file",
+                    "path": "patches/git-lfs-2.2.1",
```

Comment:
Looks weird to put some arbitrary binary in the git repository. Does it work for all architectures? Can we build it from source?

Reply:
I [believe you can](https://github.com/git-lfs/git-lfs). Looks like it needs Go which could well be a bit of a can of worms.

Reply:
I tried to but table flips were involved. Also seems overkill for such a small utility. It's 64-bit, which SparkleShare requires.

Reply:
Can you download it as part of build process instead? Also if x86_64 is the only supported architecture, you need to create `flathub.json` [file](https://github.com/flathub/com.valvesoftware.Steam/blob/master/flathub.json) to make sure it's only build on that.

Reply:
Added the flathub.json file.

I thought this was a neat solution and transparent for maintainers as git does the install work rather than hacking up something in bash and autotools. You don't want binaries in the manifest repo?

Reply:
It is possible to use the `simple` buildsystem to bypass autotools/cmake assumption, you can see example [here](https://github.com/flathub/com.jagex.RuneScape/blob/master/com.jagex.RuneScape.json#L105).